source/lib/cdk-textract-stack.ts

```diff
@@ -73,14 +74,27 @@ export class CdkTextractStack extends cdk.Stack {
     };
 
     // S3 buckets
+    const logsS3Bucket = new s3.Bucket(
+      this,
+      this.resourceName("LogsS3Bucket"),
+      {
+        bucketName: this.resourceName("logs-s3-bucket"),
+        accessControl: s3.BucketAccessControl.LOG_DELIVERY_WRITE,
+        versioned: false,
+        encryption: BucketEncryption.S3_MANAGED
+      }
+    );
+
     const documentsS3Bucket = new s3.Bucket(
       this,
       this.resourceName("DocumentsS3Bucket"),
       {
         bucketName: this.resourceName("document-s3-bucket"),
         versioned: false,
         cors: [corsRule],
-        encryption: BucketEncryption.S3_MANAGED
+        encryption: BucketEncryption.S3_MANAGED,
+        serverAccessLogsBucket: logsS3Bucket,
+        serverAccessLogsPrefix: "documentsS3Logs"
```

Comment:
We already have 2 names for this bucket  ( DocumentsS3Bucket and document-s3-bucket)
Can we reuse the one of these name as the prefix?

Reply:
Updated prefix